libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/ReplaceDialog.cs

```diff
@@ -57,7 +57,7 @@ public override async Task<DialogTurnResult> BeginDialogAsync(DialogContext dc,
             var boundOptions = BindOptions(dc, options);
 
             // set the activity processed state (default is true)
-            dcState.SetValue(TurnPath.ACTIVITYPROCESSED, this.ActivityProcessed);
+            dcState.SetValue(TurnPath.ACTIVITYPROCESSED, this.ActivityProcessed.GetValue(dcState));
```

Comment:
>, this.ActivityProcessed.GetValue(dcState) [](start = 55, length = 42)

I think I already fixed this, you should merge with master, but yes, this was wrong :)